### ISSUE TITLE
loss is nan, stopping training!

### ISSUE THREAD

**Issue by Haoqing-Wang** [LayneH/GreenMIM#6]
Wonderful job! When I try to train the Swin-B for 800 epochs, I meet this problem, 'loss is nan, stopping training'. But I find the loss value has no question. If I skip this error, the loss will keep as nan forever. Would you have some suggestions for this problem? Thanks very much!

**Comment by LayneH** [comment]
Hi, there might be multiple reasons for loss becoming NaN, e.g., AMP.
I did not encounter this problem in my pre-training experiments, so I am not sure what the problem is.
There are similar issues raised in the repo of MAE, you might want to check if they help (https://github.com/facebookresearch/mae/issues/42).

**Comment by Haoqing-Wang** [comment]
Thanks a lot!
